docker/ol-importbot-start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+python --version
+scripts/manage-imports.py --config "$OL_CONFIG" $IMPORTBOT_OPTS import-all
```

Comment:
Let's drop this env variable; currently unused.

```suggestion
scripts/manage-imports.py --config "$OL_CONFIG" import-all
```